context/app/static/js/pages/search/Search.jsx

```diff
@@ -53,7 +53,9 @@ function Search(props) {
     // Search results fields to display in table:
     resultFields,
     // Default hitsPerPage is 10:
-    hitsPerPage: 20,
+    hitsPerPage: 18,
```

Comment:
Huh: The tile view seems to be one short of a multiple of three... maybe the results are getting truncated, somehow?

<img width="402" alt="Screen Shot 2020-10-23 at 1 50 28 PM" src="https://user-images.githubusercontent.com/730388/97037114-d885d980-1536-11eb-99a7-a0389c1b52d6.png">

Reply:
Can you send me the link? I can't reproduce. Was this on the first page?

Reply:
We can't reproduce this issue.